### DISCUSSION TITLE
How do I disable the notification `Some options have changed, please run ' : Neotree migrations ' to see the changes`

### DISCUSSION THREAD

**Discussion by hyper-dot** [nvim-neo-tree/neo-tree.nvim#1144] [Q&A] [answered]
I recently updated the neotree and It is bothering every time I open neovim.

**Comment by cseickel** [comment, marked as answer]
I think it is pretty self explanatory. Why don't you run
```
:Neotree migrations
```

and make the changes to your config that it is asking you to?

**Reply by hyper-dot** [reply]
Thank you :heart:
